### DISCUSSION TITLE
New users confirmed from example.com instead of my domain

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#104] [Q&A] [answered]
I registered a new user and waited for the email confirmation. The confirmation email came but seemingly not from my site. It came from 'example.com'.  

In the email_confirmation_message.txt the variables site_name and site_domain don’t seem to have values. Emails arrive with ‘example.com’ for both variables.

Where to set site_name and site_domain?

**Comment by swiftugandan** [comment, marked as answer]
* Is this a local install or a docker install?
* Not sure what is causing it, but I think the workaround may be to change the site domain in the admin UI see below:

<img width="1409" alt="Screenshot 2021-03-22 at 16 39 38" src="https://user-images.githubusercontent.com/724629/112026052-bae08e80-8b2d-11eb-8926-4046635be212.png">

**Reply by Dan1ell** [reply]
It is a docker installation. Setting the site domain through the Django admin fixed it.

**Comment by mgogoulos** [comment]
as @swiftugandan suggests, set the Site through the Django administration
